db/seeds/posts/privacy-policy.html

```diff
@@ -0,0 +1,65 @@
+<h2 id="how-we-use-and-protect-your-information">How we use and protect your information</h2>

```

Comment:
Can we run a search-replace `&rsquo;` => `'` on this file? They don't need to be fancy.

Reply:
No problem - change has been made